docs/source/private-data/private-data.md

```diff
@@ -272,80 +272,350 @@ data can be bound to the same conditions as regular channel state data, specific
-   policy to be the buying organization and regulator), writes the private details to the
-   buyer's private data collection, and potentially deletes the private details from seller's
-   collection. Prior to final endorsement, the endorsing peers ensure private data is
-   disseminated to any other authorized peers of the seller and regulator.
-
-7. The seller submits the transaction with the public data and private data hashes
-   for ordering, and it is distributed to all channel peers in a block.
-
-8. Each peer's block validation logic will consistently verify the endorsement policy
-   was met (buyer, seller, regulator all endorsed), and verify that public and private
-   state that was read in the chaincode has not been modified by any other transaction
-   since chaincode execution.
-
-9. All peers commit the transaction as valid since it passed validation checks.
-   Buyer peers and regulator peers retrieve the private data from other authorized
-   peers if they did not receive it at endorsement time, and persist the private
-   data in their private data state database (assuming the private data matched
-   the hashes from the transaction).
-
-10. With the transaction completed, the asset has been transferred, and other
-    channel members interested in the asset may query the history of the public
-    key to understand its provenance, but will not have access to any private
-    details unless an owner shares it on a need-to-know basis.
-
-The basic asset transfer scenario could be extended for other considerations,
-for example the transfer chaincode could verify that a payment record is available
-to satisfy payment versus delivery requirements, or verify that a bank has
-submitted a letter of credit, prior to the execution of the transfer chaincode.
-And instead of transactors directly hosting peers, they could transact through
-custodian organizations who are running peers.
+
+
+## Example scenario: Asset transfer using private data collections
+
+The private asset transfer smart contract demonstrates how an asset can be represented and traded between organizations in a Hyperledger Fabric blockchain channel, while keeping details of the asset and transaction private.
+Each on-chain asset is a non-fungible token (NFT) that represents a specific asset having certain immutable metadata properties (such as size and color) with a unique owner. When the owner wants to sell the asset, both parties need to agree to the same price before the asset is transferred. The private asset transfer smart contract enforces that only the owner of the asset can transfer the asset. In the course of this tutorial, you will learn how Fabric features such as state based endorsement, private data, and access control come together to provide secured transactions that are both private and verifiable.
+
+### Scenario requirements
+
+The private asset transfer scenario is bound by the following requirements:
```

Comment:
if this whole page relates to specifically `secured-agreement` sample, we should provide the link somewhere on this page : https://github.com/hyperledger/fabric-samples/tree/master/asset-transfer-secured-agreement/chaincode-go

my main motivation being that this page  titled `private data collections` is easily confused with the `asset-transfer-private-data`  sample, being worked on here https://github.com/hyperledger/fabric-samples/pull/205

Reply:
Good point, I added back the original scenario section and added a new section for this tutorial (including a link).

Reply:
As suggested by others, converting to a separate tutorial.